.github/workflows/ci.yml

```diff
@@ -19,12 +19,12 @@ jobs:
   test:
     uses: dvsa/.github/.github/workflows/php-library-tests.yml@v3.2.3
     with:
-      php-versions: "[\"8.0\", \"8.1\"]"
+      php-versions: "[\"7.4\", \"8.0\", \"8.1\"]"
```

Comment:
Do we need php 7 here?

Reply:
Just added it for coverage because the main composer.json file uses 7.4 as well